eclass/databases.eclass

```diff
@@ -0,0 +1,429 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: databases.eclass
+# @MAINTAINER:
+# Anna <cyber+gentoo@sysrq.in>
+# @AUTHOR:
+# Anna <cyber+gentoo@sysrq.in>
+# @SUPPORTED_EAPIS: 8
+# @BLURB: eclass to test databases
```

Comment:
I suppose "test packages against databases" would be more correct.

Reply:
Missed in v2, resolved here